src/chains/ethereum/transaction/src/signing.ts

```diff
@@ -232,14 +245,24 @@ export const computeInstrinsicsFeeMarketTx = (
       encodedData.length + encodedSignature.length
     )
   ]);
+
+  const data = Buffer.concat([
+    typeBuf,
+    digest([encodedData.output], encodedData.length)
+  ]);
+  const senderPubKey = _ecdsaRecover(
```

Comment:
Are we not bypassing our checks on the chainId portion of the signature for these?